examples/megacli.cpp

```diff
@@ -3316,6 +3316,40 @@ void exec_cp(autocomplete::ACState& s)
 
 void exec_du(autocomplete::ACState& s)
 {
+    if (s.extractflag("-all"))
+    {
+        bool breakdown = s.extractflag("-breakdown");
+        struct counters { m_off_t filecount = 0, filebytes = 0, fileversioncount = 0, fileversionbytes = 0, foldercount = 0; };
+        map<Node*, counters> values;
+        for (auto& n : client->nodes)
+        {
+            auto& c = values[breakdown ? n.second->firstancestor() : nullptr];
+            switch (n.second->type)
+            {
+                case FILENODE:
+                {
+                    bool isversion = n.second->parent && n.second->parent->type == FILENODE;
```

Comment:
isversion should be `const`

Reply:
it's a local and its scope is only the next two lines, and it's obviously not changed.  I don't think we gain anything by specifying that?

Reply:
But perhaps using `isVersion` :P